### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by jjzazuet**
- Removing Cassandra specific configuration attributes documentation.
- Adding consistency level configuration documentation.
- Changing schema initialization queries for integration test cases and example application.

### CODE REVIEW

File: docs/src/docs/asciidoc/guides/cassandra.adoc [hunk 1]
```diff
@@ -37,6 +37,13 @@ Spring Session, which is in turn backed by Cassandra.
 When a Cassandra cluster is available, the primary configuration properties are available via
 `@EnableCassandraHttpSession`
 
+=== Application context provided dependencies.
+
+In order to enable Spring session's session management capabilities using Cassandra it is mandatory for your application
+to configure and auto-wire a bean of the type `com.datastax.driver.core.Session` which must define a cluster contact
+point host, as well as a port number. When injected, this bean will be used in conjunction with
+`org.springframework.cassandra.core.CqlOperations` to issue queries against the Cassandra cluster.
+
```
Comment (fitzoh) [inline docs/src/docs/asciidoc/guides/cassandra.adoc:45]: This is a bit out of date now. spring-session doesn't do anything with the host/contact points, it just needs a `CqlOperations` bean.

**Comment by fitzoh** [comment]
:+1:

**Comment by jjzazuet** [comment]
@Fitzoh updated. Does it read OK now? Thx!

**Comment by jjzazuet** [comment]
@Fitzoh , does this commit need any other changes? Cheers!

**Comment by fitzoh** [comment]
Nope @jjzazuet, just forgot to click the button :D

**Pull request merged**